packages/common/__tests__/events/nftCreationTx.ts

```diff
@@ -0,0 +1,181 @@
+/**
```

Comment:
Should we delete the file in the wallet service package that is exactly this one?

https://github.com/HathorNetwork/hathor-wallet-service/blob/master/packages/wallet-service/events/nftCreationTx.ts

Reply:
Yes, removed in 25bb3a319d0606b08076e033720f6b493e2d56b0